agent.go

```diff
@@ -717,6 +723,20 @@ func (p *pod) runContainerProcess(cid, pid string, terminal bool, started chan e
 	return nil
 }
 
+func reapChildProcesses() error {
+	var ws unix.WaitStatus
+	var rus unix.Rusage
+
+	pid, err := unix.Wait4(0, &ws, 0, &rus)
+	if err != nil {
+		return err
+	}
+
+	agentLog.Infof("Process %d reaped", pid)
```

Comment:
I wonder if there is any value in logging the `WaitStatus` (and maybe `Rusage`) here?